### ISSUE TITLE
Review handling of versions in task collection

### ISSUE THREAD

**Issue by tcompa** [fractal-analytics-platform/fractal-web#163]
While playing around with fractal-web, I noticed that the task-collection endpoint apparently ignores the task-package version, see for instance
```
Create task colleciton action
{
  package: 'fractal-tasks-core',
  version: '0.10.0a1',
  python_version: '3.9',
  package_extras: 'fractal-tasks'
}
Task collection created {
  data: {
    status: 'pending',
    package: 'fractal-tasks-core',
    venv_path: '.fractal/fractal-tasks-core0.9.4',
    task_list: [],
    log: null,
    info: 'Collecting tasks in the background. GET /task/collect/1 to query collection status'
  },
  timestamp: '2023-06-13T06:52:53.843404',
  id: 1
}
```

**Comment by tcompa** [comment]
The issue is simply that we renamed `TaskCollectPip.version` to `TaskCollectPip.package_version`, and fractal-web needs to be updated.

**Comment by tcompa** [comment]
This now works as expected (as of deab7b9), that is, we can correctly collect task packages with different versions.
The only issue which is left is that there is still an "Unspecified" version in the table:

![Screenshot from 2023-06-13 09-16-43](https://github.com/fractal-analytics-platform/fractal-web/assets/3862206/b3652b86-165f-4cc1-b6dd-c5052144382f)

**Comment by tcompa** [comment]
My bad, fixed with 3d08d30.
